app/form_models/publishers/job_application/tag_form.rb

```diff
@@ -1,8 +1,14 @@
 class Publishers::JobApplication::TagForm
   include ActiveModel::Model
   include ActiveModel::Validations
+  include ActiveModel::Attributes
 
-  attr_accessor :job_applications
+  VALID_STATUSES = %w[submitted unsuccessful reviewed shortlisted interviewing].freeze
+
+  attribute :status, :string
+  attr_accessor :job_applications, :origin, :validate_status
 
   validates_length_of :job_applications, minimum: 1
+  validates_presence_of :status, if: -> { validate_status }
+  validates_inclusion_of :status, in: VALID_STATUSES, if: -> { status.present? }
```

Comment:
If validate_status is false then would this allow us to set status to nil? Is that ok?

Reply:
yes that's fine as when we are on the tag page we don't know which status will be pick so it can be null.
when we perform the update_tag action the status has to be present and validated